example/produce_xsec_for_ZA_tanb_cosba.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/env python
+
+import math
+from cp3_llbb.Calculators42HDM.Calc2HDM import *
+import numpy as np
+import json
+import math
+
+# Alex's PhD thesis parameters
+mode = 'H'
+sqrts = 13000
+type = 2
+mh = 125
+MH = 500 
+MA = 200
+outputFile = "out_tb_cba.dat"
+ZtollBR =  3.3658 * 2 / 100. # no taus
+
+results = {
+    'tb': [],
+    'cba': [],
+    'sigma': [],
+    'sigma_errIntegration': [],
+    'sigma_err_muRm': [],
+    'sigma_err_muRp': [],
+    'BR': []
+    }
+
+def float_to_str(x, digits=2):
+    tmp = ':.{:d}f'.format(digits)
+    tmp = ('{' + tmp + '}').format(x)
+    return tmp.replace('.', 'p')
+
+#for tb in xrange(0, 50, 1):
+#    for cba in xrange(-1, 1, 5):
+for tb in np.arange(0.01, 10, 0.05):
```

Comment:
as discussed this is inefficient as we will most likely plot tan(b) in log scale in the end... maybe a double loop like this would be better ?
```python
for tb_exponent in np.arange(-2, 3, 1):
  for tb_coeff in np.arange(0, 10, 1):
    tb = tb_coeff * pow(10, tb_exponent)
(...)
```